x-pack/.i18nrc.json

```
@@ -73,6 +73,7 @@
     "xpack.serverlessSearch": "plugins/serverless_search",
     "xpack.serverlessSecurity": "plugins/serverless_security",
     "xpack.securitySolution": "plugins/security_solution",
+    "xpack.ess_security": "plugins/ess_security",
```

The plugin will be renamed to `security_solution_ess` soon. Could we set the translation prefix to:
```suggestion
    "xpack.securitySolutionEss": "plugins/ess_security",
```
In advance, so we don't have to change it later?